brave/brave-core
# move PublisherBanner to mojo type

Fixes brave/brave-browser#4613.

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [x] Linux (fails tests fixed by #2827)

## Test Plan:
1. Launch browser.
2. Visit a site with a verified publisher that has a banner (on rewards staging: kjozwiakstaging.github.io; on production: brave.com)
3. Open rewards extension and click "send a tip".
4. Notice that the custom banner shows correctly and the browser does not crash.

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
